### DISCUSSION TITLE
请求增加随机浏览闪念/笔记功能

### DISCUSSION THREAD

**Discussion by yihanglee-cn** [blinkospace/blinko#195] [Ideas]
可以随机浏览过去时间的闪念/笔记（因为每日回顾只能回顾当天的？）

**Comment by blinko-space** [comment]
这个功能感觉没有必要把？随机浏览的话 自己去翻看之前的笔记不就可以了吗

**Comment by yihanglee-cn** [comment]
> 这个功能感觉没有必要把？随机浏览的话 自己去翻看之前的笔记不就可以了吗

如果记录数量过多的话，就会很痛苦的，我现在用memos，一年不到，就已经有1000+的记录了。当数量很多的时候，我觉得随机看看以前的记录，可以当做有效回顾的一种手段。当然只是个人需求。

**Comment by StrayDogA8** [comment]
我是觉得功能不错，但是特意打开工具软件来随机回顾（而不是针对性检索）这个使用场景有点奇怪，可能每天早晨随机推几条到微信之类的常用聊天软件来培养回顾习惯会更有用些。

**Comment by ringcrl** [comment]
强烈支持添加随机回顾，目前没有真正用起来 blinko 的原因是缺乏像 flomo 那样每天随机24条回顾的能力。
目前已经习惯了每天起床之后打开 flomo 随机回顾，看以前的笔记经常会涌现出新的灵感，这是个非常好的能力，如果要自己去翻以前的笔记，现实情况就是不会去翻的。 @blinko-space 

---
The reason for adding random reviews is strongly supported. The reason why blinko is not really used is the lack of the ability to randomly 24 reviews per day like flomo.
Now I am used to turning on flomo after getting up every day to review it randomly. When reading previous notes, new inspiration often emerges. This is a very good ability. If you want to read previous notes by yourself, the reality is that you won’t read them. @blinko-space

**Comment by pudarrell2022** [comment]
我之前也是用flomo的，现在用blinko主要是用来记碎片化的学习笔记，如果能有随机回顾的话，就能复习记过的知识，温故知新。

---
I used flomo before, but now I use blinko mainly to take fragmented learning notes. If I can review it randomly, I can review the knowledge I have remembered and learn the new.

**Comment by blinko-space** [comment]
好的 这个功能会在后续版本推出

---
OK This feature will be released in subsequent versions
